### PR TITLE
Reduce the size of //tensorflow/tools/pip_package:simple_console_windows

### LINKED ISSUE #22390: bazel absence of ZIP64 support cause tensorflow build fail
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10 1803
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: v1.11.0-rc1
- **Python version**: 3.6.5
- **Bazel version (if compiling from source)**: 0.15.2
- **GCC/Compiler version (if compiling from source)**: vs2017 15.8 / cl.exe 19.15.26726
- **CUDA/cuDNN version**: 9.2.148.1/7.2.1
- **GPU model and memory**: 1080ti 11GB
- **Exact command to reproduce**: 
```
bazel build --config=opt //tensorflow/tools/pip_package:build_pip_package
bazel-bin\tensorflow\tools\pip_package\build_pip_package C:/tmp/tensorflow_pkg
```
### Describe the problem

If build artifact large than 4GB, it can't generate the python package.

A easy way to reproduce the issue
```
Please note that each additional compute capability significantly increases your build time and binary size. [Default is: 3.5,7.0]: 3.0,3.2,3.5,5.0,5.2,5.3
```

related link:
https://github.com/tensorflow/tensorflow/issues/20332#issuecomment-415974623
https://github.com/tensorflow/tensorflow/issues/22382
https://github.com/bazelbuild/bazel/blob/0.17.1/third_party/ijar/zip.cc#L74

### Source code / logs
```
Uncompressed input jar has size ???, which exceeds the maximum supported output size 4294967295.
Assuming that ijar will be smaller and hoping for the best.

Unzipping simple_console_for_windows.zip to create runfiles tree...
[./bazel-bin/tensorflow/tools/pip_package/simple_console_for_windows.zip]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of ./bazel-bin/tensorflow/tools/pip_package/simple_console_for_windows.zip or
        ./bazel-bin/tensorflow/tools/pip_package/simple_console_for_windows.zip.zip, and cannot find ./bazel-bin/tensorflow/tools/pip_package/simple_console_for_windows.zip.ZIP, period.

```

### PR DESCRIPTION
**Pull request by meteorcloudy**
This change reduce the size of `//tensorflow/tools/pip_package:simple_console_windows`'s zip file from 1000027677 bytes to 47690474 bytes for a CPU build. For GPU build, it will avoid going over 4GB when multiple CUDA compatibility are specified.

To fix https://github.com/tensorflow/tensorflow/issues/22390

/cc @gunan

### CODE REVIEW

**Review by gunan — DISMISSED** [review]

File: configure.py [hunk 1]
```diff
@@ -1421,6 +1421,13 @@ def set_windows_build_flags(environ_cp):
   # TODO(pcloudy): Remove this flag when upgrading Bazel to 0.16.0
   # Short object file path will be enabled by default.
   write_to_bazelrc('build --experimental_shortened_obj_file_path=true')
+  # When building zip file for some py_binary and py_test targets, don't
+  # include it's dependencies. This is for:
```
Comment (angersson) [inline configure.py]: Nit: "its"

File: configure.py [hunk 2]
```diff
@@ -1421,6 +1421,13 @@ def set_windows_build_flags(environ_cp):
...
+  #   1. Run python tests against system installed TF pip package.
```
Comment (angersson) [inline configure.py]: Nit: "Running python tests against the"

File: configure.py [hunk 3]
```diff
@@ -1421,6 +1421,13 @@ def set_windows_build_flags(environ_cp):
...
+  #   2. Avoid redundant files in //tensorflow/tools/pip_package:simple_console_windows,
```
Comment (angersson) [inline configure.py]: Nit: "Avoiding"

**Review by angerson — CHANGES_REQUESTED** [review]
I pointed out some small typos. Thanks for this!

File: configure.py (hunk 1 above)
Comment (meteorcloudy) [reply configure.py]: Done

File: configure.py (hunk 2 above)
Comment (meteorcloudy) [reply configure.py]: Done.

File: configure.py (hunk 3 above)
Comment (meteorcloudy) [reply configure.py]: Done

**Comment by meteorcloudy** [comment]
Thanks for the review!

**Comment by gunan** [comment]
@meteorcloudy could you send this internally instead?

**Comment by meteorcloudy** [comment]
Will do it!

**Pull request closed without merging**